Packs/Legacy/Integrations/Flashpoint/Flashpoint.yml

```diff
@@ -1,48 +1,51 @@
+category: Data Enrichment & Threat Intelligence
```

Comment:
Can't understand the changes here, most of them seem irrelevant, please download the file from demisto and upload it again